### ISSUE TITLE
Code highlight issue

### ISSUE THREAD

**Issue by ayaankhan98** [TheAlgorithms/website#92]
Codes in languages like `Python`, `Javascript`, `Dart`, `Go` etc are displayed in proper highlited format but `C++` implementations are in black and white mode meaning no code highlighting is there.

### Python
![python](https://user-images.githubusercontent.com/43348292/121563676-489a8e80-ca38-11eb-9773-5c8084ae3633.png)

### C++
![C++](https://user-images.githubusercontent.com/43348292/121563701-4f290600-ca38-11eb-8c3b-29ca53685397.png)

**Comment by Panquesito7** [comment]
For me, it's looking fine.
![image](https://user-images.githubusercontent.com/51391473/121565878-866cc800-c9e2-11eb-80d5-1f8ae6b072e7.png)

**Comment by ayaankhan98** [comment]
For me it's not working. Systems details are mentioned below
```
OS: Manjaro Linux x86_64 
Kernel: 5.4.123-1-MANJARO
Browser: Google Chrome Version 91.0.4472.77 (Official Build) (64-bit)

**Comment by ayaankhan98** [comment]
> For me, it's looking fine.
> ![image](https://user-images.githubusercontent.com/51391473/121565878-866cc800-c9e2-11eb-80d5-1f8ae6b072e7.png)

It seems like you are viewing different file, please check breadth first search algorithm

**Comment by maltejur** [comment]
Yes for breadth first search I also have the issue

**Comment by ayaankhan98** [comment]
> Yes for breadth first search I also have the issue

I do face other issues with this website also, such as most of the times links does not work. I keep cliking on links loading gets completed instantaneously but nothing happens.

**Comment by maltejur** [comment]
This seems to be a issue with highlight.js. I've updated it to the newest version, and now everything looks fine again

**Comment by maltejur** [comment]
> I do face other issues with this website also, such as most of the times links does not work. I keep cliking on links loading gets completed instantaneously but nothing happens.

Does it happen on specific links or just all of them?

**Comment by ayaankhan98** [comment]
> > I do face other issues with this website also, such as most of the times links does not work. I keep cliking on links loading gets completed instantaneously but nothing happens.
> 
> Does it happen on specific links or just all of them?

all the links

**Comment by ayaankhan98** [comment]
> This seems to be a issue with highlight.js. I've updated it to the newest version, and now everything looks fine again

Nice :+1:

**Comment by ayaankhan98** [comment]
> This seems to be a issue with highlight.js. I've updated it to the newest version, and now everything looks fine again

Thanks, but it's good to always have proper PR even for any minor change.

**Comment by maltejur** [comment]
> I do face other issues with this website also, such as most of the times links does not work. I keep cliking on links loading gets completed instantaneously but nothing happens.

Are there any errors in the console?

**Comment by ayaankhan98** [comment]
> > I do face other issues with this website also, such as most of the times links does not work. I keep cliking on links loading gets completed instantaneously but nothing happens.
> 
> Are there any errors in the console?

I will open a proper issue in regard with that after debugging the bug properly. For now I am closing this as this issue seems to be fixed by a direct commit (`0cba53db4bcc9b7132f9b33de18a489cde5d0876`) on master.
